test/txsim/blob.go

```diff
@@ -40,10 +40,10 @@ func NewBlobSequence(sizes, blobsPerPFB Range) *BlobSequence {
 	}
 }
 
-// WithNamespace provides the option of fixing a predefined namespace for
+// WithNamespaces provides the option of fixing a set of predefined namespaces for
 // all blobs.
-func (s *BlobSequence) WithNamespace(namespace share.Namespace) *BlobSequence {
-	s.namespace = namespace
+func (s *BlobSequence) WithNamespaces(namespaces []share.Namespace) *BlobSequence {
```

Comment:
[no change needed] seems technically API breaking because exported method renamed but I couldn't find any usage outside this repo so seems fine

Reply:
Yeah it's dangling so i figured nbd